main/src/cgeo/geocaching/activity/ActivityMixin.java

```diff
@@ -62,8 +63,10 @@ public static void showProgress(final AppCompatActivity activity, final boolean
             return;
         }
 
-        activity.setSupportProgressBarIndeterminateVisibility(show);
-
+        View progressBar = activity.findViewById(R.id.progress_bar);
```

Comment:
I would prefer to not create a utility method here. Showing/hiding a view is a simple one-liner so a utility method is too overwhelming IMHO.

Reply:
didn't create it but modified the existing (broken) utility method. What should happen to that one? Leaving it broken is no good, so either update or remove